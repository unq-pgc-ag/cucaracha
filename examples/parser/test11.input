fun main() {

}

fun mal() : Vec {

}
