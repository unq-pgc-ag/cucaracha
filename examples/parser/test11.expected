(Program
  (Function
    main
    Unit
    (Block
    )
  )
  (Function
    mal
    Vec
    (Block
    )
  )
)
